inc/impact.class.php

```diff
@@ -1173,7 +1176,8 @@ private static function checkIcon(string $icon_path): string {
       }
 
       // Check if icon exist on the filesystem
-      if (file_exists(__DIR__ . "/../$icon_path")) {
+      $file_path = __DIR__ . "/../$icon_path";
```

Comment:
You should use GLPI_ROOT to be sure that path will not be broken if class is moved in a namespace in the future.
```suggestion
      $file_path = GLPI_ROOT . "/$icon_path";
```